back/engines/commercial/analysis/spec/lib/auto_tagging_task_spec.rb

```diff
@@ -158,7 +158,7 @@
       it 'recudes the inputs to fit in the context window' do
         stubbed_context_window = 1000
         expect_any_instance_of(Analysis::LLM::GPT41)
-          .to receive(:context_window)
+          .to receive(:usable_context_window)
           .at_least(:once)
           .and_return(stubbed_context_window)
```

Comment:
Wonder if it might be worth stubbing the `headroom_ratio` and writing a test specifically for that>